index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="utf-8">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header>
+    <div class="header">
+      <a href="./index.html"><img src="images/logo.png"></a>
+      <button class="btn login" onclick="location.href='login/index.html'">로그인</button>
+    </div>
+  </header>
+
+  <div class="landing">
+    <section>
+      <div>
+        <h1>일상의 모든 물건을<br>거래해 보세요</h1>
+        <button class="btn" onclick="location.href='items/index.html'">구경하러 가기</button>
+      </div>
+      <img src="images/Img_home_top.png">
+    </section>
+  </div>
+
+  <main>
+    <section>
+      <div class="content-container">
+        <img src="images/Img_home_01.png">
+        <div class="description">
+          <div class="title">Hot item</div>
+          <div class="container">
+            <p>인기 상품을<br>확인해 보세요</p>
+            <p>가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="content-container reverse">
+        <img src="images/Img_home_02.png">
+        <div class="description">
+          <div class="title">Search</div>
+          <div class="container">
+            <p>구매를 원하는<br>상품을 검색하세요</p>
+            <p>구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="content-container">
+        <img src="images/Img_home_03.png">
+        <div class="description">
+          <div class="title">Register</div>
+          <div class="container">
+            <p>판매를 원하는<br>상품을 등록하세요</p>
+            <p>어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <div class="landing bottom">
+    <section>
+      <div>
+        <h1>믿을 수 있는<br>판다마켓 중고 거래</h1>
+      </div>
+      <img src="images/Img_home_bottom.png">
+    </section>
+  </div>
+  <footer>
+    <div>
+      <div>@codeit - 2024</div>
+      <div class="link">
+        <a href="privacy/index.html">Privacy Policy</a>
+        <a href="faq/index.html">FAQ</a>
+      </div>
+      <div class="sns">
+        <a href="https://www.facebook.com/" target="_blank">
+          <img src="images/ic_facebook.png">
+        </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
      <ul class="sns">
        <li>
          <a href="https://www.facebook.com/" target="_blank">
            <img src="images/ic_facebook.png">
          </a>
        </li>
```